tavern/internal/graphql/gqlgen.yml

```diff
@@ -25,15 +25,6 @@ resolver:
 # ent package. If they match it will use them, otherwise it will new ones.
 autobind:
   - realm.pub/tavern/internal/ent
-  - realm.pub/tavern/internal/ent/file
```

Comment:
is this correct?

Reply:
Yes, all of the autobind types are just located in 'ent/'